### ISSUE TITLE
Indentation broken?

### ISSUE THREAD

**Issue by kevinsjoberg** [ocaml/tuareg#262]
I wanted to try out tuareg for OCaml development, but I'm having a hard time getting the indentation to work properly. Here's the relevant excerpt from my `init.el`:

```elisp
;; Enable Tuareg (Emacs OCaml mode).
;; See https://github.com/ocaml/tuareg
(use-package tuareg
  :defer t
  :config
  (use-package merlin
    :hook (tuareg-mode . merlin-mode)))
```

As you can see from the video below, the indentation is completely haywire.

https://user-images.githubusercontent.com/503025/129352083-6ad179af-96b4-4405-92ed-82498588777f.mov

**Comment by Fourchaux** [comment]
Only the indentation of the **comments** is wrong.
You  can try ocp-indent.
```
  (use-package ocp-indent
    :hook (tuareg-mode-hook . ocp-setup-indent))
```

**Comment by kevinsjoberg** [comment]
@Fourchaux you're right. Sorry if I came across as overly negative. I'm extremely thankful for all work put into Tuareg.🙂

Didn't know about ocp-indent. Is it compatible with Tuareg?

**Comment by mattiase** [comment]
This looks like issue #179 again.
Note that tuareg knows how to indent the comments once you've written a non-comment on a later line. Try indenting the comments again. (Not to excuse the behaviour, of course.)

**Comment by kevinsjoberg** [comment]
@mattiase indeed. Pressing M-q after typing out the next line, moves the comments in place.
